.github/workflows/federated-integ-test.yml

```
@@ -100,6 +100,16 @@ jobs:
           sudo touch /etc/vbox/networks.conf
           sudo sh -c "echo '* 192.168.0.0/16' > /etc/vbox/networks.conf"
           sudo sh -c "echo '* 3001::/64' >> /etc/vbox/networks.conf"
+      - name: Cache magma test
```

To make it clear that it caches the box and not test results:
```suggestion
      - name: Cache magma-test-box
```

---

agree, but ...
* needs to be changed in .github/workflows/lte-integ-test.yml to in order to be consistent
* needs to be changed caching for other boxes as well in order to be consistent (more NIT)
* there is a task that is hopefully started soon, for using a vagrant cloud login via secret - then all of these clumpy caching entries become obsolete

---

See also: https://github.com/magma/magma/pull/13132